docs/src/SUMMARY.md

```diff
@@ -13,6 +13,14 @@
     * [Create a JupyterLab](tutorials/my-first-notebook/create-a-jupyterlab.md)
     * [Start a Jupyter notebook](tutorials/my-first-notebook/start-a-jupyter-notebook.md)
     * [Conclusion](tutorials/my-first-notebook/conclusion.md)
+  * [My first RStudio Project](tutorials/getting-started-rstudio/README.md)
+    * [Create new RStudio Project](tutorials/getting-started-rstudio/create-project.md)
+    * [Installing packages](tutorials/getting-started-rstudio/install-packages.md)
+    * [Example analysis](tutorials/getting-started-rstudio/example-analysis.md)
```

Comment:
It might be nice for the tutorials to consistently finish with a conclusion, underlining the learning points and pointing to possible next steps.